css/modules/main-container-layout.css

```diff
@@ -335,13 +328,35 @@
     padding: 0.67857rem 0.8571rem 0.8571rem 0.9285rem;
   }
 
-  /* #virtualclassCont.congrea a.inner_bt {
-    padding: 0.6785rem 1rem 0.6428rem 1rem;
-  } */
+  html.pwaRunning #virtualclassCont.congrea #preCheckcontainer .modal {
+  background-color: #e4e4e4;
+  background: linear-gradient(to right, #bdeafe 0%,#acc3fb 100%);
+}
 
-  /* #virtualclassCont.congrea .vmchat_room_bt .inner_bt {
-    padding: 0.8125rem 0.8125rem 0.6875rem 0.8125rem;
-  } */
+  html.pwaRunning #virtualclassCont.congrea #joinClass #joinClassModal {
+    background-color: #e4e4e4 !important;
+    background: linear-gradient(to right, #bdeafe 0%,#acc3fb 100%);
+  }
+
+  html.pwaRunning #virtualclassCont.congrea .progressbar li.active.current::before {
+    color: unset;
+    background: -webkit-linear-gradient(#9c47fc, #356ad2);
+    -webkit-background-clip: text;
+    -webkit-text-fill-color: transparent;
+  }
+
+  html.pwaRunning #virtualclassCont.congrea #virtualclassPreCheck .testName {
```

Comment:
Codacy found an issue: [Missing vendor-prefixed CSS gradients for Firefox 3.6+, Old Webkit (Safari 4+, Chrome), Opera 11.1+.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5740126)